test/helpers/reactjs_helper_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class ReactjsHelperTest < ActionView::TestCase
+  include ReactjsHelper
+
+  Foreman::Plugin.register(:foreman_react) {}
+  Foreman::Plugin.register(:foreman_angular) {}
+
+  def webpack_asset_paths(bundle_name, opts)
+    ["<script src=\"https://foreman.example.com:3808/webpack/#{bundle_name}.js\"></script>"]
```

Comment:
shouldnt this be on the same url as the server in prod/test?

Reply:
Does it matter? It is just a stub for test, the method comes from  [webpack-rails](https://github.com/mipearson/webpack-rails/blob/master/lib/webpack/rails/helper.rb#L15) and I cannot use it directly in test because it complains about missing manifest.